src/meltano/core/utils/__init__.py

```diff
@@ -591,3 +591,55 @@ def safe_hasattr(obj: Any, name: str) -> bool:
     except AttributeError:
         return False
     return True
+
+
+def strtobool(val: str) -> bool:
+    """Convert a string representation of truth to true (1) or false (0).
+
+    True values are 'y', 'yes', 't', 'true', 'on', and '1'; false values
+    are 'n', 'no', 'f', 'false', 'off', and '0'.  Raises ValueError if
+    'val' is anything else.
+
+    Re-implemented from distutils.util.strtobool to avoid importing distutils.
```

Comment:
I double checked that this is the case in the code but might be a helpful hint to have also in the docstring.

```suggestion

    Case is ignored in text comparisons.

    Re-implemented from distutils.util.strtobool to avoid importing distutils.
```